canon/src/repr_host.rs

```diff
@@ -129,27 +158,19 @@ where
     }
 
     /// Retrieve a mutable value behind this representation and run a closure on it
```

Comment:
Since you got rid of the function parameter, maybe remove the part about the closure

Reply:
agreed.

Reply:
done